### PR TITLE
Rename to mercurius

### PR DESCRIPTION
**Pull request by mcollina**
renamed to mercurius! Let me know if I missed anything

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1285,6 +1285,16 @@ async function run () {
...
 
+## Acknowledgements
+
+The project is kindly sponsored by:
+
+* [NearForm](https://www.nearform.com) for [Matteo](https://github.com/mcollina)'s time in maintaining this module.
+
+The mercurius name was gracefully donated by [Marco Castelluccio](https://github.com/marco-c).
+The usage of that library was described in https://hacks.mozilla.org/2015/12/web-push-notifications-from-irssi/, and
+you can find that codebase in https://github.com/marco-c/mercurius.
```
Comment (mcollina) [inline README.md:1296]: @marco-c PTAL

Comment (marco-c) [reply README.md:1296]: Thanks ;)

**Comment by asciidisco** [comment]
the repos `About` section still contains `Fastify GraphQL adapter` 
Scrennshot 2020-09-25 um 11 27 43

**Comment by mcollina** [comment]
@asciidisco what would you recommend as text?

**Comment by asciidisco** [comment]
> @asciidisco what would you recommend as text?

I'm not exactly sure, the question is what will benefit the user most. I see it more like a `GraphQL server library based on Fastify` rather than a `Fastify GraphQL adapter`. But that's such a minor thing, I'm not sure if it's even worth the time to discusss TBH.

**Comment by mcollina** [comment]
What do you think of..

> Implement GraphQL servers and gateways with Fastify

File: examples/subscription/mqemitter-mongodb-subscription.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const GQL = require('fastify-gql')
+const mercurius = require('fastify-gql')
```
Comment (PacoDu) [inline examples/subscription/mqemitter-mongodb-subscription.js:3]:
```suggestion
const mercurius = require('mercurius')
```

or `../..` to match the other examples

File: index.d.ts [hunk 3]
```diff
@@ -239,12 +239,12 @@ export interface FastifyGQLCommonOptions {
   };
 }
 
-export type FastifyGQLOptions = FastifyGQLCommonOptions & (FastifyGQLGatewayOptions | FastifyGQLSchemaOptions)
+export type MercuriusOptions = MercuriusCommonOptions & (MercuriusGatewayOptions | MercuriusSchemaOptions)
 
 declare function fastifyGQL
```
Comment (frikille) [inline index.d.ts:244]: `fastifyGQL` should be renamed to `mercurius` for consistency

**Review by frikille — COMMENTED** [review]
In `make-resolver.js` line 10 a symbol is created with the name `fastify-gql.entity-resolvers` that should be renamed for `mercurius.entity-resolvers` for consistency.

Apart from these two changes, I think the rest is good 👍

**Comment by asciidisco** [comment]
> What do you think of..
> 
> > Implement GraphQL servers and gateways with Fastify

That sounds very descriptive to me 👍

File: index.d.ts (hunk 3 above)
Comment (mcollina) [reply index.d.ts:244]: Sorry I missed this, I'm doing this change on master.

**Pull request merged**